src/inputs/UploadFile/UploadFile.js

```diff
@@ -151,14 +164,16 @@ UploadFile.propTypes = {
      {
       button: 'string',
       invalidFileMessage: 'string',
-      label: 'string'
+      label: 'string',
+      delete: 'string'
     }
    </pre>
    */
   labels: PropTypes.shape({
     button: PropTypes.string.isRequired,
     invalidFileMessage: PropTypes.string.isRequired,
     label: PropTypes.string,
+    delete: PropTypes.string.isRequired,
```

Comment:
Adding a new key with `isRequired` in the `labels` prop could be considered a breaking change: a webapp with the previous format for `labels` using the new version of this component will cause errors to be printed in the browser console complaining about this missing key in `labels`.

We can prevent this breaking change by:
- removing `isRequired` in the key declaration
- adding a default fallback in the code: `title={ labels.delete || 'Delete file' }` (fallback values cannot be only defined in the `defaultProps` when `isRequired` is removed for the `delete` key in `labels`)

We will then have to clean this work-around in the next major version :slightly_smiling_face:

Reply:
Done